esm-samples/jsapi-custom-workers/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: {
+    index: ['./src/index.css', './src/index.js']
+  },
+  node: false,
+  output: {
+    path: path.join(__dirname, 'dist'),
+    chunkFilename: 'chunks/[id].js',
+    publicPath: '',
+    clean: true
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'dist'),
+    compress: true,
+    port: 3001,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        enforce: "pre",
+        use: ["source-map-loader"],
+      },
+      {
+        test: /\.css$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          'css-loader'
+        ]
+      },
+    ]
+  },
+  plugins: [
```

Comment:
What about adding `ignoreWarnings: [/Failed to parse source map/],` since we know it will output warnings from at least html-sanitizer.  Will reduce the number of questions we get.